### PR TITLE
Add arm32 ci testing.

### PR DESCRIPTION
**Pull request by jashook**
Note this will add a new lst file and leverage the old existing arm64
infrastructure.

@RussKeldorph @adiaaida @mmitche ptal

### CODE REVIEW

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

File: netci.groovy [hunk 1]
```diff
@@ -1078,9 +1105,36 @@ def static addTriggers(def job, def branch, def isPR, def architecture, def os,
...
-                    {
-                        Utilities.addGithubPRTriggerForBranch(job, branch, "${os} ${architecture} Cross ${configuration} Build")
+                    // Set up a private trigger
+                    def contextString = "${os} ${architecture} Cross ${configuration}"
+                    if (scenario != 'default')
+                        contextString += " ${scenario}"
+                    contextString += " Build"
+                    // Debug builds only.
+                    if (configuration != 'Debug') {
+                    contextString += " and Test"
```
Comment (pgavlin) [inline netci.groovy]: Nit: indentation is off.

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test CI Please

**Comment by jashook** [comment]
@dotnet-bot test Windows_NT Arm64 Checked

**Comment by jashook** [comment]
@dotnet-bot test ci please

**Pull request merged**